packages/flutter/lib/src/widgets/bottom_navigation_bar_item.dart

```diff
@@ -96,4 +96,9 @@ class BottomNavigationBarItem {
   ///  * [Icon.color] and [ImageIcon.color] to control the foreground color of
   ///    the icons themselves.
   final Color? backgroundColor;
+
+  /// The text to display in the tooltip for this [BottomNavigationBarItem].
+  ///
+  /// The tooltip will only appear on an item in the [BottomNavigationBar].
+  final String? toolTip;
```

Comment:
nit: tooltip seems to be a single word: https://en.wikipedia.org/wiki/Tooltip#:~:text=From%20Wikipedia%2C%20the%20free%20encyclopedia,a%20cursor%2C%20usually%20a%20pointer.
```suggestion
  final String? tooltip;
```